code/modules/mob/living/carbon/emote.dm

```diff
@@ -0,0 +1,78 @@
+/datum/emote/carbon
+	mob_type_blacklist_typecache = list(/mob/living/carbon/slime)
+
+/datum/emote/carbon/whimper
+	key = "whimper"
+	key_third_person = "whimpers"
+	message = "whimpers"
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/carbon/burp
+	key = "whimper"
```

Comment:
Whimpers when should be burps

Reply:
Oops, thanks for catching that!!

Reply:
> owrries

I like that typo for some reason. It's like owies but cuter.
Annnd is apparently related to oats somehow...